lucene/core/src/java/org/apache/lucene/search/FuzzyQuery.java

```diff
@@ -193,7 +172,7 @@ protected TermsEnum getTermsEnum(Terms terms, AttributeSource atts) throws IOExc
     if (maxEdits == 0 || prefixLength >= term.text().length()) {  // can only match if it's exact
       return new SingleTermsEnum(terms.iterator(), term.bytes());
     }
-    return new FuzzyTermsEnum(terms, atts, getTerm(), termLength, maxEdits, automata);
+    return new FuzzyTermsEnum(terms, atts, getTerm(), maxEdits, prefixLength, transpositions);
```

Comment:
do we need to update the java docs on multi term query to reflect that fuzzy query also uses the attribute source now as well?

Reply:
Good idea, will update